portfolio/src/main/java/com/google/sps/data/CalendarEventsGroup.java

```diff
@@ -0,0 +1,102 @@
+package com.google.sps.data;
+
+import com.google.common.collect.Lists;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/** This class models a group of calendar events. */
+public class CalendarEventsGroup {
+
+  // This variable that represents a group of calendar events will
+  // always be sorted in ascending order based on their start time.
+  public List<CalendarEvent> events;
```

Comment:
nit: I think we can make this final too? It seems like we only initialize it in the constructor.